src/is.test.ts

```diff
@@ -33,16 +33,319 @@ import {
   isTypedArray*/
 } from './is';
 
+// isArray(value)
 describe('isArray(value)', () => {
   it('Should be a function', () => {
     expect(isArray).to.be.a('function');
   });
 
-  it('Should should return true when "value" is type any[]', () => {
-    expect(isArray([1, '2', true])).to.equal(true);
+  it('Should return true when "value" is type any[]', () => {
+    expect(isArray([])).to.equal(true);
+    expect(isArray([1, 2, 3])).to.equal(true);
+    expect(isArray(['a', 'b', 'c'])).to.equal(true);
+    expect(isArray([1, '2', true, {}])).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type any[]', () => {
+    expect(isArray('This is a string')).to.equal(false);
+    expect(isArray(100)).to.equal(false);
+    expect(isArray(undefined)).to.equal(false);
+    expect(isArray(null)).to.equal(false);
+    expect(isArray(true)).to.equal(false);
+    expect(isArray(false)).to.equal(false);
+    expect(isArray({})).to.equal(false);
+    expect(isArray()).to.equal(false);
+  });
+});
+
+// isBoolean(value)
+describe('isBoolean(value)', () => {
+  it('Should be a function', () => {
+    expect(isBoolean).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type boolean', () => {
+    expect(isBoolean(true)).to.equal(true);
+    expect(isBoolean(false)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type boolean', () => {
+    expect(isBoolean('This is a string')).to.equal(false);
+    expect(isBoolean(100)).to.equal(false);
+    expect(isBoolean(undefined)).to.equal(false);
+    expect(isBoolean(null)).to.equal(false);
+    expect(isBoolean([])).to.equal(false);
+    expect(isBoolean({})).to.equal(false);
+    expect(isBoolean()).to.equal(false);
+  });
+});
+
+// isInfinity(value)
+describe('isInfinity(value)', () => {
+  it('Should be a function', () => {
+    expect(isInfinity).to.be.a('function');
+  });
+
+  it('Should return true when "value" is Infinity', () => {
+    expect(isInfinity(Infinity)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not Infinity', () => {
+    expect(isInfinity('This is a string')).to.equal(false);
+    expect(isInfinity(100)).to.equal(false);
+    expect(isInfinity(undefined)).to.equal(false);
+    expect(isInfinity(null)).to.equal(false);
+    expect(isInfinity(true)).to.equal(false);
+    expect(isInfinity(false)).to.equal(false);
+    expect(isInfinity([])).to.equal(false);
+    expect(isInfinity({})).to.equal(false);
+    expect(isInfinity()).to.equal(false);
+  });
+});
+
+// isInteger(value)
+describe('isInteger(value)', () => {
+  it('Should be a function', () => {
+    expect(isInteger).to.be.a('function');
+  });
+
+  it('Should return true when "value" is an Integer', () => {
+    expect(isInteger(0)).to.equal(true);
+    expect(isInteger(100)).to.equal(true);
+    expect(isInteger(100.00)).to.equal(true);
```

Comment:
Similar to Infinity, let's check for other ways to get integers:

```suggestion
    expect(isInteger(100.00)).to.equal(true);
    expect(isInteger(Number.MAX_SAFE_INTEGER)).to.equal(true);
    expect(isInteger(Number.MIN_SAFE_INTEGER)).to.equal(true);
```

Reply:
Okay. Do you want me to check other integers (like different numbers with varying magnitudes) or other inbuilt constants (like MAX_SAFE_INTEGER)?

Reply:
No I think this is good.

Reply:
okay. Then I will commit the suggestions, take a look and send the pull request for it to be merged!